west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: edttool
```

Comment:
The main branch of this repository is now mirrored, so this can be remoed.

Reply:
Excellent.